pennylane/vqe/vqe.py

```diff
@@ -115,10 +115,6 @@ def __init__(self, coeffs, observables, simplify=False, group=False, id=None):
         self._ops = list(observables)
         self._wires = qml.wires.Wires.all_wires([op.wires for op in self.ops], sort=True)
 
-        if isinstance(coeffs, list):
-            coeffs = np.array(coeffs)
-        self.data = [coeffs]
-
```

Comment:
We leave it up to the parent class to define `data` (which is important, because it is used by the tape to collect parameters!)